microsoftgraph/microsoft-graph-explorer-v4
# AB#39703 Interns/gracet9182/39703 update teams app

## Overview

Updated the app id with the deployed id. 


## Testing Instructions

* Install the deployed version of the app (Graph Explorer) for the user
* Login into that user's account in GE v4
* Ensure they have TeamsAppInstallation permissions
* Toggle and see that if they have it installed, then the pop up will not show up. Otherwise, it will show up
